interface-definitions/dhcpv6-server.xml.in

```diff
@@ -309,6 +309,18 @@
                     </properties>
                     <children>
                       #include <include/generic-disable-node.xml.i>
+                      <leafNode name="mac-address">
```

Comment:
Why not use: `#include <include/interface/mac.xml.i>`

Reply:
I was planning to take up refactoring separately. But sure, this can be added in this PR. Change coming up.